.github/CONTRIBUTING.md

```diff
@@ -211,21 +234,21 @@ This prevents nesting levels from getting deeper then they need to be.
 
 
 ### Other Notes
-* Code should be modular where possible, if you are working on a new class then it is best if you put it in a new file.
+* Code should be modular where possible; if you are working on a new addition, then strongly consider putting it in its own file unless it makes sense to put it with similar ones (i.e. a new tool would go in the "tools.dm" file)
 
 * Bloated code may be necessary to add a certain feature, which means there has to be a judgement over whether the feature is worth having or not. You can help make this decision easier by making sure your code is modular.
 
-* You are expected to help maintain the code that you add, meaning if there is a problem then you are likely to be approached in order to fix any issues, runtimes or bugs.
+* You are expected to help maintain the code that you add, meaning that if there is a problem then you are likely to be approached in order to fix any issues, runtimes, or bugs.
 
-* Do not divide when you can easily convert it to a multiplication. (ie `4/2` should be done as `4*0.5`)
+* Do not divide when you can easily do the same thing with multiplication. (ie `4/2` should be done as `4*0.5`)
 
-#### Enforced not enforced
-The following different coding styles are not only not enforced, but it is generally frowned upon to change them over from one to the other for little reason:
+#### Grammar and syntax enforcement
```

Comment:
Don't change this heading. We have alot of links that directly link to this heading, and changing it invalidates them.

Also Enforced not enforced is better